contributors.yaml

```
@@ -0,0 +1,277 @@
+- given-names: Timothy
+  family-names: Hallett
+  orcid: "https://orcid.org/0000-0002-9681-0424"
+  affiliation: "Imperial College London"
+  website: "https://www.imperial.ac.uk/people/timothy.hallett"
+  github-username: tbhallett
+  role: Joint lead epidemiology
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Andrew
+  family-names: Phillips
+  orcid: "https://orcid.org/0000-0003-2384-4807"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/5430"
+  github-username: andrew-phillips-1
+  role: Joint lead epidemiology
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Asif
+  family-names: Tamuri
+  orcid: "https://orcid.org/0000-0001-6633-3789"
+  affiliation: "University College London"
+  website: "https://www.homepages.ucl.ac.uk/~ucbtaut/"
+  github-username: tamuri
+  role: "Lead developer"
+  contributions:
+    - Software development
+- given-names: Joseph
+  family-names: Mfutso-Bengo
+  orcid: "https://orcid.org/0000-0001-9396-3700"
+  affiliation: "Kamuzu University of Health Sciences"
+  website: "https://sphfm.medcol.mw/staff/dr-joseph-mfutso-bengo-phd" # to check - link seems not to work
```

```suggestion
  website: "https://www.researchgate.net/profile/Joseph-Mfutso-Bengo"
```

As current link (taken from existing authors page) does not seem to work possibly could use ResearchGate page as a replacement?